### PR TITLE
Minor bug fixes and code cleanup

### PR DESCRIPTION
**Pull request by Rupan**
My initial thought here was to port keypatch to the IDA 7.0 beta, but happily I see that you've already started work on this front.  Upon initially loading keypatch.py in PyCharm, several import errors and code quality issues were identified.  I've changed calls to __init__ so that they use super() instead of direct calls to superclass names, added missing / rearranged imports, qualified constants, fixed invalid references, removed dead code, etc.

### CODE REVIEW

File: keypatch.py [hunk 1]
```diff
@@ -16,12 +16,16 @@
...
 import re
 import json
 from keystone import *
-
+import idc
+import idaapi
+try:
+    from idc import GetOpType, GetOpnd, ItemEnd
+except ImportError:
+    from idc_bc695 import GetOpType, GetOpnd, ItemEnd
```
Comment (aquynh) [inline keypatch.py]: where is this idc_bd695 from?

**Comment by aquynh** [comment]
thanks for this cleanup! it looks good, except my comment on the idc_bd695.

File: keypatch.py (hunk 1 above)
Comment (Rupan) [reply keypatch.py]: As you are aware, there are major API changes between 32-bit IDA 6.x and 64-bit IDA 7.0.  As a way to aid plugin porting, Hex-Rays has provided a backwards compatibility layer for some of the old APIs in 64-bit IDA 7.0.  The ```idc_bc695``` module provides these wrappers.

Comment (Rupan) [reply keypatch.py]: Certainly the code which calls these functions could be refactored and wrapped in conditional statements, but this seemed to be the cleaner approach.

**Comment by aquynh** [comment]
merged, thanks!

**Pull request merged**